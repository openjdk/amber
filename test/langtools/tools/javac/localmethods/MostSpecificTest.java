/*
 * @test /nodynamiccopyright/
 * @bug 8233038
 * @summary Tests need to be written for local methods feature
 * @compile/fail/ref=MostSpecificTest.out -XDrawDiagnostics MostSpecificTest.java
 */

public class MostSpecificTest {
    public static void main(String args []) {
        void foo(String s, Object o) {
        }
        void foo(Object o, String s) {
        }

        foo(new String(), new String()); // ambigious
    }
}
