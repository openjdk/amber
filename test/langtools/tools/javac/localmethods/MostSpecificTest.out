MostSpecificTest.java:15:9: compiler.err.ref.ambiguous: foo, kindname.method, foo(java.lang.String,java.lang.Object), main(java.lang.String[]), kindname.method, foo(java.lang.Object,java.lang.String), main(java.lang.String[])
1 error
